src/app/services/progressesApi.ts

```diff
@@ -0,0 +1,19 @@
+import { api } from './api';
+
+export const progressesApi = api.injectEndpoints({
+    endpoints: (builder) => ({
```

Comment:
headers object is missing the Authorization header, which is required for authentication.

Reply:
credentials include is the one which sets it